src/libs/getComponentDisplayName.ts

```diff
@@ -0,0 +1,10 @@
+import {ComponentType} from 'react';
+
+/**
+ * Returns the display name of a component
+ *
+ * @param component
+ */
```

Comment:
```suggestion
/** Returns the display name of a component */
```

Remove `@params` and `@returns` completely if there is no description

Reply:
removed